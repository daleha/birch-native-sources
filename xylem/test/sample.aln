;; Modified by bwf on Thu Jul 29 17:03:06 1993
;
PEADRRA
MGVFNVEDEITSVVAPAILYKALVTDADTLTPKVIDAIKSIEIVEGNGGAGTIKKLTFVEDGETKHVLHK
VELVDVANLAYNYSIVGGVGFPDTVEKISFEAKLSAGPNGGSIAKLSVKYYTKGDAAAPTEEQLKSDKAK
GDGLFKALERYCLAHPDYN*
;
; Modified by bwf on Thu Jul 29 17:03:06 1993
PEADRRG
MGVFNFEEEATSIVAPATLHKALVTDADILTPKVIDAIKSIEIVEGNGGPGTIKKLTFVEDGETKYVLHK
VELVDDANWANNYSIVGGVGLPDTVEKISFEAKLSAGPNGGSIAKLSVKYYTKGDAIPSEEE-IKNGKAK
GEGIFKALEGYCVANPDYN*
