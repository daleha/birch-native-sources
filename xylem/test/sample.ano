LOCUS       PEADRRA       728 bp ss-mRNA            PLN       15-MAR-1989
DEFINITION  Pea (P.sativum) disease resistance response protein (PI176) mRNA,
            complete cds.
ACCESSION   M18249
KEYWORDS    disease resistance response protein.
SOURCE      Pea (P.sativum, cultivar Alaska) pea pod, cDNA to mRNA.
  ORGANISM  Pisum sativum
            Eukaryota; Plantae; Embryobionta; Magnoliophyta; Magnoliopsida;
            Rosidae; Fabales; Fabaceae.
REFERENCE   1  (bases 1 to 728)
  AUTHORS   Fristensky,B., Horovitz,D. and Hadwiger,L.A.
  TITLE     cDNA sequences for pea disease resistance response genes
  JOURNAL   Plant Mol. Biol. 11, 713-715 (1988)
  STANDARD  full automatic
COMMENT     Draft entry and computer-readable sequence for [1] kindly provided
            by B.Fristensky, 25-JAN-1988.
FEATURES             Location/Qualifiers
     mRNA            1..728
                     /note="pI176 mRNA"
     CDS             66..545
                     /note="disease resistance response protein"
                     /codon_start=1
                     /translation="MGVFNVEDEITSVVAPAILYKALVTDADTLTPKVIDAIKSIEIV
                     EGNGGAGTIKKLTFVEDGETKHVLHKVELVDVANLAYNYSIVGGVGFPDTVEKISFEA
                     KLSAGPNGGSIAKLSVKYYTKGDAAAPTEEQLKSDKAKGDGLFKALERYCLAHPDYN"
BASE COUNT      217 a    132 c    143 g    236 t
ORIGIN      Unreported.
//
LOCUS       PEADRRB       594 bp ss-mRNA            PLN       15-MAR-1989
DEFINITION  Pea (P.sativum) disease resistance response protein (PI206) mRNA,
            3. end.
ACCESSION   M18250
KEYWORDS    disease resistance response protein.
SOURCE      Pea (P.sativum, cultivar Alaska) pea pod, cDNA to mRNA.
  ORGANISM  Pisum sativum
            Eukaryota; Plantae; Embryobionta; Magnoliophyta; Magnoliopsida;
            Rosidae; Fabales; Fabaceae.
REFERENCE   1  (bases 1 to 594)
  AUTHORS   Fristensky,B., Horovitz,D. and Hadwiger,L.A.
  TITLE     cDNA sequences for pea disease resistance response genes
  JOURNAL   Plant Mol. Biol. 11, 713-715 (1988)
  STANDARD  full automatic
COMMENT     Draft entry and computer-readable sequence for [1] kindly provided
            by B.Fristensky, 25-JAN-1988.
FEATURES             Location/Qualifiers
     mRNA            <1..594
                     /note="pI206 mRNA"
     CDS             <1..528
                     /note="disease resistance response protein"
                     /codon_start=1
                     /translation="VFVMLFALSSAIPNKRKPYKPCKNLVFYFHDILYNGKNAANATS
                     AIVAAPEGVSLTKLAPQSHFGNIIVFDDPITLSHSLSSKQVGRAQGFYIYDTKNTYTS
                     WLSFTFVLNSTHHQGTITFAGADPIVAKTRDISVTGGTGDFFMHRGIATITTDAFEGE
                     AYFRLGVYIKFFECW"
BASE COUNT      187 a    106 c    100 g    201 t
ORIGIN      Unreported.
//
LOCUS       PEADRRG      2290 bp ds-DNA             PLN       30-JUN-1992
DEFINITION  Pea disease resistance response protein (DRRG49-c) gene, complete
            cds.
ACCESSION   J03680
KEYWORDS    disease resistance response protein.
SOURCE      Pea DNA.
  ORGANISM  Pisum sativum
            Eukaryota; Plantae; Embryobionta; Magnoliophyta; Magnoliopsida;
            Rosidae; Fabales; Fabaceae.
REFERENCE   1  (bases 1 to 2290)
  AUTHORS   Chiang,C.C. and Hadwiger,L.A.
  TITLE     Cloning and characterization of a disease resistance response gene
            in pea inducible by Fusarium solani
  JOURNAL   Mol. Plant Microb. Interact. 3, 78-85 (1990)
  STANDARD  full automatic
COMMENT     Draft entry and printed sequence for [Mol. Plant Microb. Interact.
            (1990) In press] kindly submitted by
            C.C.Chiang, 24-JAN-1990.
FEATURES             Location/Qualifiers
     exon            <1151..1331
                     /number=1
                     /note="disease resistance response protein (DRRG49-c)"
     intron          1332..1419
                     /note="DRRG49-c intron"
     exon            1420..>1715
                     /number=2
                     /note="disease resistance response protein"
     CDS             join(1151..1331,1420..1715)
                     /note="disease resistance response protein (DRRG49-c)"
                     /codon_start=1
                     /translation="MGVFNFEEEATSIVAPATLHKALVTDADILTPKVIDAIKSIEIV
                     EGNGGPGTIKKLTFVEDGETKYVLHKVELVDDANWANNYSIVGGVGLPDTVEKISFEA
                     KLSAGPNGGSIAKLSVKYYTKGDAIPSEEEIKNGKAKGEGIFKALEGYCVANPDYN"
BASE COUNT      797 a    369 c    357 g    767 t
ORIGIN
//
