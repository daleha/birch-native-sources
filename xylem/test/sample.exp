>PEADRRA:CDS1
@M18249:66..545
>PEADRRB:CDS1
@M18250:<1..528
>PEADRRG:CDS1
@J03680:join(1151..1331,1420..1715)
