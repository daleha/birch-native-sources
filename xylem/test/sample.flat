"G-A1
010011001100110000111001010110010010010110100000111101000001
011101101110100111010111000100110001001101000111111111011010
101011100111111111000101100011011010110001011100111101001101
011111010000010101111101011000111001111000000000100110110000
100000000000000000000000101011000101000101100100011000100001
0000100000000010100
"G-A2
010010001101110100111011010100000000011110010010110001000000
010011101110100110001111001101111001101000000110010111110100
000010110101001100000000010111011110110001000101011100001111
010111000000010100111111011100111101011000101000000110001010
000000011111111001011000000000000101000101100000011100011101
0000101100001110100
"G-A3
010001001100110000111000000110101010010111001110110011000001
010011101110001110001110001101110101001001000110010111000000
0010101001111111000?????????????????????????????????????1111
011110000000010100111001011000111001011000101000100010111101
100110111101111000010000000010000101000001100100111000001101
0000101100100010100
"G-A4
000001001100110010111010001100011101010111010010110101000001
111011111110011110001110001100111001001000100111110001000110
000110111101111110000000010000011010101100100001011101001101
011010010001000100111001011010111000011000101000100110111000
000101110111111011011000000000000000000000000100011011101101
1110111100001110100
"G-A5
010011001100110000111001010100010001010111010110111011000000
000000000010100110000000000100100001001000000010110101100000
001010110111111110100101010011011010110001010100111101001101
011111000000000100111101011001111001011000101000101010111101
001100111111110001011000000000000100000001101111011100010100
0000100000000010101
"G-A6
011101011101110010111000010101011001011111001110110101000000
000001101110000110001000000100100001001001100111010111100100
001010100101001100000000010001011110001001010100111101001001
001111001000000100111001011010111000101000101010110010111000
010101111111111011010000000010000101000101101100011000001101
1010101100101100100
"G-A8
011001011110100111111001010100011001011110001010110111000???
???????????????????????????????????????100000101111101100000
100110100101000100000101101011011110001100010100111101001101
001111000101000100101000011000111001111000101000100000100011
100101110111111011011010000010000000000101101101011000001101
0010100000000000000
"G-A9
010011001100110000111001010100010001010110010000101000000100
010001100010100110010010000100111001001001100111111111000100
001010110101000100000101010011011010110001010100111101001101
011111000000000100111101011000111001011000101001100010110010
010110111111100001011000100000000000000001101100011100010101
0000100000001010101
"G-B2
010001000100110000000000000010001010010111000010111001000001
010001101011000111101010000100110001001100000010000001000000
000110100101001100001101010011011010101001011000111101001101
011110010001000100111001011000111000001000101000100010110011
000100110001110000011000000010000101000001101100011001101101
0000101101100110100
"G-B4
010001001100110000111000010110001000010111000010111001000000
110001101010000110001010000100110001001101100111110111100000
001010110111101100000100010011011000101000010000111100001101
011110010000000100101001011000111001011000101000110110111111
010100111001100000011010000011010101000101101101011011101101
0000100101101000100
"G-B5
000001001110110000011000010100001000011111100010111101000000
001000001011000111000000000100100001001100000010000001000010
100110100101000000000100110011011010110010010100111101001111
111100010001000100111101011000111100111000101000100000101010
000000110000000000001000011011100111110111110101011101101101
1010111100101010100
"G-B6
011001010110110000111000010100001000011111000000010000000000
011011101110000110001010000100111001001001100010000101000000
10011010010100000000010101101101101010100101010011110100????
????????????????01111001111000111001111000101001100000101110
000000111101100000000000001011010101000101101111011000101101
0000100100000000100
"G-C1
010001001100110000111000000100001000010111000010010001000000
010001101010000110001000000100100001001001100010010111000000
001010100111000100000101011011111010101001010000111101001101
011110000000000101111001111000111001011000101000100000101011
010111111101100000000000000010000101000001100101011000001001
0000100000000000100
"G-C3
010011000100110000110001010100001001010111010110111011000000
010011111110111111011111001101111101001001000111110111010100
001010110111111110100101011011011010110001010100111101001101
011111000000000100111101011000111001011001101000100000101011
101000010001000000001000000010000101000001101111011100010101
0000101000000000101
"G-C4
010001001100110000111000000100001000011111000010010000000000
100000001010000110001000000100100001000000000110010111000010
001110111111111110100000000011010010001010000000011100001001
011010000000000100111000011000111000011000101000100000101011
010111110111101001010110000000000100000101101101011001001101
1000100100000010100
"G-C6
010001001111110010111111100100000001010111010110101101000000
010011111110001111101110011100111001001011000111010111100110
000010100111101100000000000111011110110000000111011100001000
100001000001000000111010011000111000011000101000100000100010
000001111111100000001000000000000000000000001100011101001101
1000101101000110101
"G-C8
010011001101110000111001010101000000011110010010111111000???
???????????????????????????????????????000000011011111100100
00111011011111111000010110011101111001000100011011110000????
????????????????00111111011000111001011000101000100001111011
111100111111100000001000000000000100000001101111011100010111
0000100000000010100
"G-C9
000001000010110010111000000100001000010111000110101111000000
000000001010000100101000000000000001001110100111110111010010
001000110111100010100100011001011010101000000000011100001101
011110000001000100111000011000111100011000001100100111110111
000111110001100000011010001001000101000101101101011001100101
0000101001000000100
"G-C10
000001000110110010111000010110001000010111001100101100000000
010001101010000110101010000000010001001101100011110101000010
001000100101001100000000001101011010101000010000111101001101
011110000001000100111001011000111100011000101000100000111011
100111111001101001011000000000000101000101101101011001100101
0000100001001000100
"G-D2
010001001100111000111000000101011000010111000000010001000000
110011101111000111011010000100111001001001100010110111010000
001010110101100100000101010011011010101001010000111100001111
011110010000000100111101011000111000011000001000100000101011
001000111001110000000000000011000101000001101100011000011101
0000101100100000100
"G-D3
000011101100110000111011010101000111010110010011101101000001
001000000010100100000000000100000001000100010010010001000000
000010100100010110010001100011011000010001000011011111011101
011111000001000100111101011110111010011000101000101010101011
001000011111000010000000000000000100000001100101011100110001
0010000000000000101
"G-D4
010011000100110000111001010100000000010110010011101001000001
110011101110100110010110011100111001101011000001000101110100
000010100101110110100100010111011010101000000010111100001111
010011000001000100111100011000111010011000001000100000110101
100110011111101000001000000000000100001000100101111101010101
1010100100000000001
"G-D5
010001010100111000011000010110001000010010000010110011000000
000000001011000100100000001000100001000100000110011001000010
001110100101100010000101100011011100101001010000111101101101
011100010001000100111000011000111000011000001000100000100101
000100111011101000011010001011000101010101100100111000101101
0000100000001100100
"G-D6
010001000100111000101000100101000000010110000000010000000000
000000001010000101000000000100100001000001000110010101000000
001010100101100110000101000011011010110001010000111101001101
111110000000000100111001011000111000111000001000100000101011
010110111101101000011000000000000101000001100100011100011101
1000100100000000100
"G-D7
011001010110100110111000010100011100011110000011010111000000
000000001011000100000000000100000001001000000010010001000000
000010100101110100000101110011011010101001010110111101101111
011110000001000100111001011000111101111000101001100000101011
110100111111111000010010101011010101011101100100011000001101
0000100000000001100
"G-D8
000001000010110000000000100101011101111001010110101111000001
110011101111001111110111101101111101101100000101000111101010
001100111101111010100001101110010001101000100100111111001101
111100010001000100111101011000111100111000101001100010100011
001000111110111000011010010011100101000101100100011001001001
1000000000000011001
"G-E1
011001001110110010111000110100011000111110000000011100000011
110011111110001110101110001100111001101000010010010001010000
010110100101011010000001000011011010101001000000011110011101
011111010000000100111001111000111101111000101000100110111111
110100111001010000000000000000000100000111100100110000001000
0000000001000000100
"G-E2
????????????????????????????00010110010110010000001100000001
010110111010010110101110001100010001000000000000001000001000
000110100101010111000000000010011000001000010000111010010110
011111000001000101011000011010110000001000001000100110110011
101000111010010000001000000000000000000001000110010000001000
0000000101000000100
"G-E4
110011000100110000111000110100011000011111010010011101100000
111010101111100101101110001100111001001100110111010101100010
000110101101100100000000000010010000001000000001111101001101
111111000001000000111100011010111000011000001000100000100101
001000010011000000001000001011011101000101101101010001001000
0000000000000000000
"G-E7
000001000100111000111000010111011000011110101010110101000000
111011101110011111011111001100111101001101000111110111000110
101010100101100100000101010011011010101011111100111101011001
111011010000000100111101011000111001011000001000100010101011
110111111101111001111010100010000000000001100101111000101101
1011100100101101110
"G-E9
010011001100110000101000010100001100010110000010110001000001
000000001010100110000000000100100001000000000010000000000000
000010100100111111000101000011011010101001010100111101011101
011010010000000000111001011000111001011000101000100110101011
000000111101010000000000000000000000000001000101011000010000
0000000000000010101
"G-R1
010001000100110000111000000100001000010111000000010000000000
000000001010000100000000000100100001000000100110010101000000
001010100101000010101101111011011010101001010000111101001101
011110000000000100111001011000111000011000001000100000101011
110100111101111000000010100000000000000001100101011000001101
0000100000000000100
"G-R2
010001001100110000111000000100001100010110000011010011000001
000000001010000110000000000100100001000000000010010001010000
001010100101111111100000010011011010101001000000111100001101
011110000000000100111001011000111001011000101100100010111011
011100110101110000000010001010000100010001100100011000001000
0000000100000010110
"G-R3
010001001100110000111000000100001000010111000011010001000000
000000001010000100000000000000000000000001000010010000000000
000010100100010010000101010011011010101001010000111101011101
011110000000010100111001011000111001011000001000100010101011
000100111101101000000010000010000101000001100101011000001101
0000110000100100100
"G-R4
010001000100110000111000000100001100010111000111010011000000
000000001010000100000000001100100001000000100110100101000000
001010110111000110000001010011011010101001000000111100001101
011110000000000100111001011000111001011000001000100000101011
001101110001000000000010001011000101010001100101010000001000
0000000000000000100
"G-R5
010010001101110010101001100100000000011110110011010000100001
110000101010100110001110010100111101101000000010000000001000
000010100000000000000001100111011110010001100111011111011101
011001000001000000111100011000111000011000011000100010101111
00100100000000000000000100000000010000100010????????????????
???????????????????
"G-R6
000001001100110000011000010010000100010000100110010111000000
111000001110001111000111001101111101011100000100010000000000
101000100100111110100111100101011101101011000100111111011011
111000110001010100111100011000111100011100101000100110111011
001000000000000000001000001011000011100101101111011000100101
1010010100000001111
"G-R7
010011000100110000110010110100000000011110010111101111101000
010010110110101110001110001100101101101000000010000000000000
000010100100000000000000000000011000001000000010011100011101
010111000000000100111001011100111101011000101000101010100101
000001011111010000001001000000000100001101100100010000010100
0010000000001011000
"G-R8
010011001100110000111000010111100100010110010110110111010011
110011111110111110010111001111101101101001000100010111010000
000010110111111111000001110011011010001001010000101101001001
011011001000010110111100011010111000001000101000100110101011
100100111101111011000000100010000100000001100100100000100001
0000100000001011111
"G-R9
000001000100110000011000010010000000110000100000010101000000
111010101110000111010110001100111001011100000100110101011010
101010100100000000000000110000011101101010000100111111011001
111000010001000100111100011000111100011110101000100110111011
010100110000111000001000000011000010000101100100011000100001
1010100100001111100
"G-R10
010001000100110000111001010100010000010111010000001001100001
110011101110001110001110011100110001001001100111110101110110
001010110111000000000101111111011010110000010100011101001101
011011000001000100111100011000111100011000111000100110111010
000110111111100000011010001001000101010000100000011101010101
0010100000000000100
