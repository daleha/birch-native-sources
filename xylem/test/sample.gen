LOCUS       PEADRRA       728 bp ss-mRNA            PLN       15-MAR-1989
DEFINITION  Pea (P.sativum) disease resistance response protein (PI176) mRNA,
            complete cds.
ACCESSION   M18249
KEYWORDS    disease resistance response protein.
SOURCE      Pea (P.sativum, cultivar Alaska) pea pod, cDNA to mRNA.
  ORGANISM  Pisum sativum
            Eukaryota; Plantae; Embryobionta; Magnoliophyta; Magnoliopsida;
            Rosidae; Fabales; Fabaceae.
REFERENCE   1  (bases 1 to 728)
  AUTHORS   Fristensky,B., Horovitz,D. and Hadwiger,L.A.
  TITLE     cDNA sequences for pea disease resistance response genes
  JOURNAL   Plant Mol. Biol. 11, 713-715 (1988)
  STANDARD  full automatic
COMMENT     Draft entry and computer-readable sequence for [1] kindly provided
            by B.Fristensky, 25-JAN-1988.
FEATURES             Location/Qualifiers
     mRNA            1..728
                     /note="pI176 mRNA"
     CDS             66..545
                     /note="disease resistance response protein"
                     /codon_start=1
                     /translation="MGVFNVEDEITSVVAPAILYKALVTDADTLTPKVIDAIKSIEIV
                     EGNGGAGTIKKLTFVEDGETKHVLHKVELVDVANLAYNYSIVGGVGFPDTVEKISFEA
                     KLSAGPNGGSIAKLSVKYYTKGDAAAPTEEQLKSDKAKGDGLFKALERYCLAHPDYN"
BASE COUNT      217 a    132 c    143 g    236 t
ORIGIN      Unreported.
        1 tacacaacta ggcaagcaat ttcttagttc tttctcacac tttagcaata taaaacaata
       61 tcatcatggg tgtttttaat gttgaagatg aaatcacttc tgttgtagca cctgctatac
      121 tctacaaagc tctagttaca gatgctgaca ctcttactcc aaaggttatt gatgccatca
      181 aaagtatcga aattgttgaa ggaaacggag gtgctggaac catcaagaaa ctcactttcg
      241 ttgaagatgg tgaaaccaaa catgtgttgc acaaagtgga gttagtagat gttgctaact
      301 tggcttacaa ctatagcata gttggtggtg ttggatttcc agacacagtt gagaagatct
      361 catttgaggc taaactgtct gcaggaccaa atggaggatc cattgcaaag ctgagtgtga
      421 aatattacac aaaaggtgat gctgctgctc ctactgaaga gcaactcaag agtgacaaag
      481 ctaagggtga tggtcttttc aaggctcttg agcgttactg tttggctcat cctgattaca
      541 actaaactat ataatcatca acaagtgtgt gttgttatgt atactatata atcatcatgt
      601 gtgcttaatt tggctgccaa tgtaatcccc ctgttttgtt ttttcccttt cccttttctt
      661 tgtggactaa aagtgtgaga ttgtaagtca tgtatacctc tccttcaata aattataatc
      721 atttatat
//
LOCUS       PEADRRB       594 bp ss-mRNA            PLN       15-MAR-1989
DEFINITION  Pea (P.sativum) disease resistance response protein (PI206) mRNA,
            3. end.
ACCESSION   M18250
KEYWORDS    disease resistance response protein.
SOURCE      Pea (P.sativum, cultivar Alaska) pea pod, cDNA to mRNA.
  ORGANISM  Pisum sativum
            Eukaryota; Plantae; Embryobionta; Magnoliophyta; Magnoliopsida;
            Rosidae; Fabales; Fabaceae.
REFERENCE   1  (bases 1 to 594)
  AUTHORS   Fristensky,B., Horovitz,D. and Hadwiger,L.A.
  TITLE     cDNA sequences for pea disease resistance response genes
  JOURNAL   Plant Mol. Biol. 11, 713-715 (1988)
  STANDARD  full automatic
COMMENT     Draft entry and computer-readable sequence for [1] kindly provided
            by B.Fristensky, 25-JAN-1988.
FEATURES             Location/Qualifiers
     mRNA            <1..594
                     /note="pI206 mRNA"
     CDS             <1..528
                     /note="disease resistance response protein"
                     /codon_start=1
                     /translation="VFVMLFALSSAIPNKRKPYKPCKNLVFYFHDILYNGKNAANATS
                     AIVAAPEGVSLTKLAPQSHFGNIIVFDDPITLSHSLSSKQVGRAQGFYIYDTKNTYTS
                     WLSFTFVLNSTHHQGTITFAGADPIVAKTRDISVTGGTGDFFMHRGIATITTDAFEGE
                     AYFRLGVYIKFFECW"
BASE COUNT      187 a    106 c    100 g    201 t
ORIGIN      Unreported.
        1 gtttttgtga tgttgtttgc tttaagttca gcaattccaa acaagagaaa gccatataaa
       61 ccatgcaaaa acctagtctt ttattttcat gacatacttt acaatggaaa gaatgcagca
      121 aatgcaacat cagcaatagt agcagctcca gaaggtgtta gtttaactaa attggcacct
      181 caatcccact ttggtaacat aatagttttt gatgacccta tcacattaag ccatagcctt
      241 tcttcaaaac aagttggaag agcacaaggg ttttatattt atgataccaa aaacacatac
      301 acttcttggc ttagtttcac ttttgttctt aatagcactc atcatcaagg aaccattact
      361 tttgctggag ctgacccaat tgtcgccaaa actagagata tttctgtcac tggtggtact
      421 ggagatttct ttatgcatag aggaattgct actattacca ctgatgcctt tgaaggcgag
      481 gcttattttc gacttggtgt ttatatcaag ttctttgagt gttggtaact atcaaattaa
      541 gtactactta ctataataaa agcaattaaa tttgaagtta aattattgtt gtct
//
LOCUS       PEADRRG      2290 bp ds-DNA             PLN       30-JUN-1992
DEFINITION  Pea disease resistance response protein (DRRG49-c) gene, complete
            cds.
ACCESSION   J03680
KEYWORDS    disease resistance response protein.
SOURCE      Pea DNA.
  ORGANISM  Pisum sativum
            Eukaryota; Plantae; Embryobionta; Magnoliophyta; Magnoliopsida;
            Rosidae; Fabales; Fabaceae.
REFERENCE   1  (bases 1 to 2290)
  AUTHORS   Chiang,C.C. and Hadwiger,L.A.
  TITLE     Cloning and characterization of a disease resistance response gene
            in pea inducible by Fusarium solani
  JOURNAL   Mol. Plant Microb. Interact. 3, 78-85 (1990)
  STANDARD  full automatic
COMMENT     Draft entry and printed sequence for [Mol. Plant Microb. Interact.
            (1990) In press] kindly submitted by
            C.C.Chiang, 24-JAN-1990.
FEATURES             Location/Qualifiers
     exon            <1151..1331
                     /number=1
                     /note="disease resistance response protein (DRRG49-c)"
     intron          1332..1419
                     /note="DRRG49-c intron"
     exon            1420..>1715
                     /number=2
                     /note="disease resistance response protein"
     CDS             join(1151..1331,1420..1715)
                     /note="disease resistance response protein (DRRG49-c)"
                     /codon_start=1
                     /translation="MGVFNFEEEATSIVAPATLHKALVTDADILTPKVIDAIKSIEIV
                     EGNGGPGTIKKLTFVEDGETKYVLHKVELVDDANWANNYSIVGGVGLPDTVEKISFEA
                     KLSAGPNGGSIAKLSVKYYTKGDAIPSEEEIKNGKAKGEGIFKALEGYCVANPDYN"
BASE COUNT      797 a    369 c    357 g    767 t
ORIGIN
        1 gtcgacacat atgtaaacag gtcggcctat agtaaatcta ataaatcctc aagatatatg
       61 taactactaa tcttgatgtc aacatttctc cacacatctt gatttgactc accagtcttg
      121 ttaccctagt gaagaaatca acgatgcttt caccttcttg ttcctcactc gttactagag
      181 tgattcaacc acaccttgat tggaaaatgg tttcacctca caatgatttg aaaagaaaag
      241 aattatgtga aagtgaattg agaaagataa gagaaattag gatttaggga gaaagagaag
      301 aagattgatc aaattctgca gattttctct ttgtgtttta ttccaactga acttccattc
      361 tttttaacta ctttacaaca gtaagttacc tcgctattta tagatgatat tacttacaca
      421 cctagtaatc tcgaactaat tcaactaata aatactaatt tttaactttg acgaggttaa
      481 ctccttcgac aactacttac ttcaattgtt taagactata ttcgactatt aatttatcga
      541 atacattcga ctttgacttt attgaatctt acatacatgc ttaacacaaa taattacata
      601 tttaactttt ttttttctcc atctctcttt actatattaa ataatgtgaa ataatataaa
      661 tactgtttaa catgttttat ttcaatatcc caatactgac ctcaaactca aaatttagaa
      721 gaaaaaagaa ctagcaacaa cataatttgt atgaatagac gtaacctagt ttttcacaaa
      781 ataaccagtt aaatttaact gtttatatat ttaagtgata atcatttatc tcggattctt
      841 tactcgacac aaatgagtta agatgagtat atgttagttt tggtaaaaaa tgtataattt
      901 agttagatat atgtttaatt tgagtaggat atgttggcac gcacgaacca aatacgtata
      961 ggaacgtgga taaaaataat actattgacc aaatgcatat cagaacgtga atacaaaatg
     1021 gtccagcgac ttggtattaa tagctataca atgcatcttc atttcttata aatagaactt
     1081 caactccact cgtaaatcac acagctaggc aagcaccttc ttatttatag cattataaat
     1141 catcattatc atgggtgttt tcaattttga ggaagaagcc acttccattg tagctcctgc
     1201 tacacttcac aaagctctgg ttacagatgc tgacattctt actccaaagg ttattgatgc
     1261 catcaaaagt attgaaattg ttgaaggaaa cggtggcccc ggaaccatca agaaactcac
     1321 tttcgttgaa ggtcagtatt agaatattct tttcatgata tatcattatt actagtatta
     1381 ttgtttatga ttgtgaaatg aatcaattgt gtgttgcaga cggtgaaacc aagtatgtgt
     1441 tacacaaagt ggagttagta gatgatgcta actgggcaaa caactacagc atagttggag
     1501 gtgttggact tccggacaca gttgagaaga tctcgtttga ggctaaattg tctgcaggac
     1561 caaatggagg atccattgca aagctgagtg tgaaatatta taccaaaggt gatgctattc
     1621 ctagtgaaga ggaaatcaag aatggcaaag ccaaaggtga aggtattttc aaggctcttg
     1681 aaggttactg tgtggctaat cctgattaca actaaaaaat ttaattaagt gagtgcttgt
     1741 tttattatgg tgtgttatga cacattttat tgcatctgtc ggcttaattt gttttcttat
     1801 ttttctttct tttccctttc ctattgttga ggaagtgtga gtttgagatt gtaagtcatg
     1861 tttgtaccac gttttaagaa attataataa tacgtatgtt cttttttata tatttttttt
     1921 tctctagaaa atattgttac taaaacaaaa ttgttactaa aaaacattta aaatattggt
     1981 agggatttga taccaccaat gtaaaccatt gaaatgaaga tccaaaacgt gactactttg
     2041 attgttttgg taataaaata aaaaatttca atcacaccaa aaaaatttaa tttgtatata
     2101 actaaaataa ataatcttct acagaacaat aacaagataa aatacaaaaa aacacaataa
     2161 tttgtgtact cagttcgatc aaaaaaatct actctaaagg taagaaaaac tctgagtcat
     2221 tatacttcct attgtgggcc gaccaaagtc aatccaagag tccaaccaga gtcagaccaa
     2281 tcaaggatcc
//
