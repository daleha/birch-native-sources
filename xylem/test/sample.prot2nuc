PROT2NUC       Version  8/10/94

     IUPAC-IUP AMINO ACID SYMBOLS
     [J. Biol. Chem. 243, 3557-3559 (1968)]

          Phe         F          Leu         L          Ile         I
          Met         M          Val         V          Ser         S
          Pro         P          Thr         T          Ala         A
          Tyr         Y          His         H          Gln         Q
          Asn         N          Lys         K          Asp         D
          Glu         E          Cys         C          Trp         W
          Arg         R          Gly         G          STOP        *
          Asx         B          Glx         Z          UNKNOWN     X


     IUPAC-IUB SYMBOLS FOR NUCLEOTIDE NOMENCLATURE
     [Cornish-Bowden (1985) Nucl. Acids Res. 13: 3021-3030.]

     Symbol         Meaning              | Symbol         Meaning
     ------------------------------------+---------------------------------
     G              Guanine              | k              G or T
     A              Adenine              | s              G or C
     C              Cytosine             | w              A or T
     T              Thymine              | h              A or C or T
     U              Uracil               | b              G or T or C
     r              Purine (A or G)      | v              G or C or A
     y              Pyrimidine (C or T)  | d              G or T or A
     m              A or C               | n              G or A or T or C

pI39
              5             10             15             20             25
M  E  K  K  S  L  A  A  L  S  F  L  L  L  L  V  L  F  V  A  Q  E  I  V  V  
ATGGArAArAArTCnCTnGCnGCnCTnTCnTTyCTnCTnCTnCTnGTnCTnTTyGTnGCnCArGArAThGTnGTn
            AGyTTr      TTrAGy   TTrTTrTTrTTr   TTr                        

             30             35             40             45             50
T  E  A  N  T  C  E  H  L  A  D  T  Y  R  G  V  C  F  T  N  A  S  C  D  D  
ACnGArGCnAAyACnTGyGArCAyCTnGCnGAyACnTAyCGnGGnGTnTGyTTyACnAAyGCnTCnTGyGAyGAy
                        TTr            AGr                     AGy         

             55             60             65             70
H  C  K  N  K  A  H  L  I  S  G  T  C  H  D  W  K  C  F  C  T  Q  N  C  
CAyTGyAArAAyAArGCnCAyCTnAThTCnGGnACnTGyCAyGAyTGGAArTGyTTyTGyACnCArAAyTGy
                     TTr   AGy                                          

