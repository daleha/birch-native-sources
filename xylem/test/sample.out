>PEADRRA:CDS1
atgggtgtttttaatgttgaagatgaaatcacttctgttgtagcacctgc
tatactctacaaagctctagttacagatgctgacactcttactccaaagg
ttattgatgccatcaaaagtatcgaaattgttgaaggaaacggaggtgct
ggaaccatcaagaaactcactttcgttgaagatggtgaaaccaaacatgt
gttgcacaaagtggagttagtagatgttgctaacttggcttacaactata
gcatagttggtggtgttggatttccagacacagttgagaagatctcattt
gaggctaaactgtctgcaggaccaaatggaggatccattgcaaagctgag
tgtgaaatattacacaaaaggtgatgctgctgctcctactgaagagcaac
tcaagagtgacaaagctaagggtgatggtcttttcaaggctcttgagcgt
tactgtttggctcatcctgattacaactaa
>PEADRRB:CDS1
gtttttgtgatgttgtttgctttaagttcagcaattccaaacaagagaaa
gccatataaaccatgcaaaaacctagtcttttattttcatgacatacttt
acaatggaaagaatgcagcaaatgcaacatcagcaatagtagcagctcca
gaaggtgttagtttaactaaattggcacctcaatcccactttggtaacat
aatagtttttgatgaccctatcacattaagccatagcctttcttcaaaac
aagttggaagagcacaagggttttatatttatgataccaaaaacacatac
acttcttggcttagtttcacttttgttcttaatagcactcatcatcaagg
aaccattacttttgctggagctgacccaattgtcgccaaaactagagata
tttctgtcactggtggtactggagatttctttatgcatagaggaattgct
actattaccactgatgcctttgaaggcgaggcttattttcgacttggtgt
ttatatcaagttctttgagtgttggtaa
>PEADRRG:CDS1
atgggtgttttcaattttgaggaagaagccacttccattgtagctcctgc
tacacttcacaaagctctggttacagatgctgacattcttactccaaagg
ttattgatgccatcaaaagtattgaaattgttgaaggaaacggtggcccc
ggaaccatcaagaaactcactttcgttgaag
acggtgaaaccaagtatgtgttacacaaagtggagttagtagatgatgct
aactgggcaaacaactacagcatagttggaggtgttggacttccggacac
agttgagaagatctcgtttgaggctaaattgtctgcaggaccaaatggag
gatccattgcaaagctgagtgtgaaatattataccaaaggtgatgctatt
cctagtgaagaggaaatcaagaatggcaaagccaaaggtgaaggtatttt
caaggctcttgaaggttactgtgtggctaatcctgattacaactaa
